src/main/java/org/apache/sling/jcr/repoinit/impl/NodePropertiesVisitor.java

```diff
@@ -16,31 +16,37 @@
  */
 package org.apache.sling.jcr.repoinit.impl;
 
-import java.util.List;
+import java.util.ArrayList;
 import java.util.Calendar;
+import java.util.List;
 
-import javax.jcr.Session;
 import javax.jcr.Node;
 import javax.jcr.PathNotFoundException;
-import javax.jcr.Value;
 import javax.jcr.PropertyType;
 import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+import javax.jcr.Value;
 
+import org.apache.jackrabbit.api.security.user.Authorizable;
+import org.apache.jackrabbit.util.Text;
 import org.apache.jackrabbit.value.BooleanValue;
 import org.apache.jackrabbit.value.DateValue;
 import org.apache.jackrabbit.value.DoubleValue;
 import org.apache.jackrabbit.value.LongValue;
 import org.apache.jackrabbit.value.StringValue;
-
 import org.apache.sling.repoinit.parser.operations.PropertyLine;
 import org.apache.sling.repoinit.parser.operations.SetProperties;
+import org.jetbrains.annotations.NotNull;
 
 /**
  * OperationVisitor which processes only operations related to setting node
  * properties. Having several such specialized visitors makes it easy to control
  * the execution order.
  */
 class NodePropertiesVisitor extends DoNothingVisitor {
+    private static final String PATH_AUTHORIZABLE = ":authorizable:";
```

Comment:
Maybe add a comment that the colons are added by the repoinit parser for an `authorizable(name)` expression, like for `home(name)` and similar notations which the parser transforms to `:authorizable:name#`.

And I think if we go ahead with this PR we should add a test that uses `authorizable(name)` to the parser's test suite ([test-34.txt](https://github.com/apache/sling-org-apache-sling-repoinit-parser/blob/master/src/test/resources/testcases/test-34.txt) and test-99.txt) to expose that syntax there, even if it's transparent for the parser.

Reply:
I'll add a comment blurb before those constants to provide some details.

Regarding new tests, those would be in a different project, so I added subtask [SLING-10252](https://issues.apache.org/jira/browse/SLING-10252) for tracking that.